networkconfig/config.go

```diff
@@ -67,3 +72,17 @@ func (n NetworkConfig) SlotsPerEpoch() uint64 {
 func (n NetworkConfig) GetGenesisTime() time.Time {
 	return time.Unix(int64(n.Beacon.MinGenesisTime()), 0)
 }
+
+// RSAMessageFork returns epoch for RSA message fork
+func (n NetworkConfig) RSAMessageFork(currentEpoch spec.Epoch) bool {
+	switch n.Name {
+	case HoleskyStage.Name:
+		return true
+	case Mainnet.Name:
+		return currentEpoch >= MainnetRSAForkEpoch
+	case TestNetwork.Name:
+		return currentEpoch >= TestNetworkRSAForkEpoch
+	default:
+		return false
+	}
+}
```

Comment:
Instead of having constants and this func, I think we can just add an `RSAForkEpoch` field to each network. In networks which don't have this fork, we can put `math.MaxUint64`.

Reply:
done